src/assets/styles/global.scss

```diff
@@ -45,6 +45,13 @@
   overflow: hidden;
 }
 
+.calcite-match-height {
```

Comment:
Just wondering, Is there a way in tailwind to define a utility class? Just wondering if we should be using tailwind here at all since I don't really know it well :)

Reply:
Also, how are we going to document these utility classes? Should we have a `.mdx` page for storybook we can use to document them. At least a `markdown` file somewhere?

Reply:
re: doc
I think that's a good idea.

re: tailwind
I was thinking that, but so far it seems like the Tailwind utility classes aren't accessible if you're just using the components. But mayhapz I'm missing something.

Reply:
IMO `global.scss` should not store an entire utility class set. It's really more about global things that all components need, ie. theme, hydration, etc.

Reply:
Where could I put this so that it can be used in the way demonstrated in the [demo app](https://github.com/Esri/calcite-components/pull/1929/files#diff-cad01ac18aa67c8ff5e4a590929fee4211f25f2476e955c46cb2c6aa490195c8R117-R120)?
Assuming that there's not something weird about it.

Reply:
Also, yes...the demo app seems kinna weird, but it's a reality.

Reply:
Maybe we should discuss all the options at a meeting?

- component to do this?
- utility class? 
- Just have them apply the needed styles?